src/api/models/User.ts

```diff
@@ -110,6 +111,10 @@ User.init(
         bio: {
             type: DataTypes.STRING,
             allowNull: false
+        },
+        favoriteActivities: {
+            type: DataTypes.ARRAY(DataTypes.INTEGER),
+            allowNull: true
```

Comment:
allowNull would result in error when you call getFavoriteActivities (length of `null` is `undefined`). i manually modified the DB to replace all `null`s with empty arrays. you should use defaultValue: [] instead

Reply:
changed it to defaultValue:[]